lisa/tools/ethtool.py

```diff
@@ -300,6 +300,16 @@ def get_device_channels_info(
         )
 
         device_channel_info = DeviceChannel(interface, result.stdout)
+
+        # Find the vCPU count to accurately get max channels for the device.
+        result = self.node.execute(
```

Comment:
If there is no special reason, use `lscpu` to get cpu_count.

Reply:
Done